<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>576</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>576</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Avalon Miner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>51</y>
      <width>494</width>
      <height>484</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_status">
     <attribute name="title">
      <string>Status</string>
     </attribute>
     <widget class="QLabel" name="label_hashrate">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>20</y>
        <width>191</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Hashrate Details</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_pool">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>190</y>
        <width>191</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Pool Details</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>230</y>
        <width>470</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>32</number>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Pool</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Active</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Last Commit</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_start">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>380</y>
        <width>152</width>
        <height>48</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>48</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
      </property>
      <property name="text">
       <string>Start Mining</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_av">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>58</y>
        <width>152</width>
        <height>94</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/hashratebg.png)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_cur">
      <property name="geometry">
       <rect>
        <x>326</x>
        <y>58</y>
        <width>153</width>
        <height>94</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/hashratebg.png)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_20s">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>58</y>
        <width>153</width>
        <height>94</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/hashratebg.png)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_GHS_20s">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>70</y>
        <width>153</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>GHS 20s</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_20s">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>110</y>
        <width>153</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0 Gh/s</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_av">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>110</y>
        <width>153</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0 Gh/s</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_GHS_av">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>70</y>
        <width>153</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>GHS av</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_cur">
      <property name="geometry">
       <rect>
        <x>326</x>
        <y>110</y>
        <width>153</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0 Gh/s</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_GHS_cur">
      <property name="geometry">
       <rect>
        <x>326</x>
        <y>70</y>
        <width>153</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>GHS cur</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_settings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox_port1">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
       </widget>
      </item>
      <item row="15" column="4">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="13" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_name3">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_port3">
        <property name="text">
         <string>Pool Port:</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_pass3">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_port2">
        <property name="text">
         <string>Pool Port:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_name3">
        <property name="text">
         <string>Worker Name:</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_pass3">
        <property name="text">
         <string>Worker Password:</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_port1">
        <property name="text">
         <string>Pool Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_url1">
        <property name="text">
         <string>Pool URL:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_pass1">
        <property name="text">
         <string>Worker Password:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_url2">
        <property name="text">
         <string>Pool URL:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_name1">
        <property name="text">
         <string>Worker Name:</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_name2">
        <property name="text">
         <string>Worker Name:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_pass2">
        <property name="text">
         <string>Worker Password:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_url3">
        <property name="text">
         <string>Pool URL:</string>
        </property>
       </widget>
      </item>
      <item row="15" column="2">
       <widget class="QPushButton" name="pushButton_save">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
        <property name="text">
         <string>  Save  </string>
        </property>
       </widget>
      </item>
      <item row="15" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="11" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_url3">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_pass2">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_name2">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_url2">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_pass1">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_name1">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_url1">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_info1">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1st Pool Information:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_info2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>2nd Pool Information:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QLabel" name="label_info3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>3rd Pool Information:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="spinBox_port2">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QSpinBox" name="spinBox_port3">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:6px;padding:2px 2px;</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_graphics">
     <attribute name="title">
      <string>Hashrate</string>
     </attribute>
     <widget class="QWidget" name="widget_cur" native="true">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>471</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_status">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>494</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>494</width>
      <height>36</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>494</width>
      <height>36</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/notworkingbg.png)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>488</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NOT MINING</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButton_start</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>graphicsView_status</tabstop>
  <tabstop>graphicsView_20s</tabstop>
  <tabstop>graphicsView_av</tabstop>
  <tabstop>graphicsView_cur</tabstop>
  <tabstop>lineEdit_url1</tabstop>
  <tabstop>lineEdit_name1</tabstop>
  <tabstop>lineEdit_pass1</tabstop>
  <tabstop>lineEdit_url2</tabstop>
  <tabstop>lineEdit_name2</tabstop>
  <tabstop>lineEdit_pass2</tabstop>
  <tabstop>lineEdit_url3</tabstop>
  <tabstop>lineEdit_name3</tabstop>
  <tabstop>lineEdit_pass3</tabstop>
  <tabstop>pushButton_save</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
